addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -7075,6 +7075,9 @@ registry.SnippetSave = SnippetOptionWidget.extend({
                                     const defaultSnippetName = _.str.sprintf(_t("Custom %s"), this.data.snippetName);
                                     const targetCopyEl = this.$target[0].cloneNode(true);
                                     delete targetCopyEl.dataset.name;
+                                    for (let tabIDTargetCopyEl of targetCopyEl.querySelectorAll('[id]')){
+                                        tabIDTargetCopyEl.id = "";
```

Comment:
- Use `const` instead of `let` when you can :+1: 
- This could use a comment like "Removing the ids is especially important for tab snippets which blablabla"
- You have a linter error here, you forgot a space between `)` and `{` of the `for`. You should install a linter to tell it to you. There is a new tooling for that https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#tooling I am actually not using that myself yet (still using my old linter) and I am not sure how it works for old versions actually (probably try the installation by going to the `master` branch first). Let's make you the guinea pig :smile: If you don't manage to install that as explained in the wiki, tell me and we'll have a look together :+1: